platform/lib/config.js

```diff
@@ -28,6 +28,20 @@ const GROW_OUT_DIR = utils.project.absolute('platform/pages');
 
 const ENV_DEV = 'development';
 const ENV_PROD = 'production';
+const AVAILABLE_LOCALES = [
```

Comment:
We should then remove all available locales from the pod spec to avoid duplication.